### PR TITLE
Term Entry: Design Sprint

### PR DESCRIPTION
**Pull request by ishg-153**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by HishamT — COMMENTED** [review]
Hey @ishg-153, I have reviewed your entry and left some comments. 

P.S. the picture is very impressive! 😃

File: content/uiux/concepts/design-methodologies/terms/design-sprint/design-sprint.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+![The Five Phases of Design Sprint](https://raw.githubusercontent.com/Codecademy/docs/main/media/design-sprint-phases)
+
+## About Design Sprint
+
+Design Sprint allows the team to quickly and efficiently conceptualize solutions to problems and validate them before integrating into the product.
+
+Another design methodology similar to Design Sprint is [Design Thinking](https://www.codecademy.com/resources/docs/uiux/design-methodologies/design-thinking). The major difference between the two is that Design Sprint (as the name suggests) is used to explore solutions with speed before investing resources into development. Additionally, Design Thinking is an iterative process while Design Sprint is not.
+
+## Additional Resources
```
[the remark this answers is not in the document]
Comment (ishg-153) [reply content/uiux/concepts/design-methodologies/terms/design-sprint/design-sprint.md:43]: It isn't really possible to give an example of design sprint, so including the article may help. Plus, I saw existing UIUX entries also add external resources like [this one](https://github.com/Codecademy/docs/blob/main/content/uiux/concepts/attitudinal-research/attitudinal-research.md).

**Comment by ishg-153** [comment]
@HishamT made suggested changes let me know about the additional resources. Also, there are other instances where `Design Sprint`, `Design Thinking`, etc are capitalized (which are not first mentions nor will they be converted to links) so should those be lower cases too.

**Review by HishamT — APPROVED** [review]
LGTM! I've decided we can keep the additional resources section.

**Pull request merged**